plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -426,6 +443,7 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 	s->input = input ? bstrdup(input) : NULL;
 	s->input_format = input_format ? bstrdup(input_format) : NULL;
 	s->is_hw_decoding = obs_data_get_bool(settings, "hw_decode");
+	s->ffmpeg_options = ffmpeg_options ? bstrdup(ffmpeg_options) : NULL;
```

Comment:
This allocation seems to be missing a corresponding `bfree` call.

Reply:
this should be fixed.  I can't believe that I missed that